scripts/easyarty/functions/fn_postInit.sqf

```diff
@@ -0,0 +1,64 @@
+if (isDedicated) exitWith {};
+easyArtyRunning = false;
```

Comment:
heavily suggesting the use of proper prefixing
easyarty_running
easyarty_pfh
easyarty_displayEh

this means it becomes less problematic moving the script to other missions or even a mod some time in the future